lib/3scale_toolbox/commands/service_command.rb

```diff
@@ -29,6 +30,7 @@ def self.command
       add_subcommand(Delete::DeleteSubcommand)
       add_subcommand(Create::CreateSubcommand)
       add_subcommand(Apply::ApplySubcommand)
+      add_subcommand(CopySubcommand)
```

Comment:
This does not seem consistent with the other commands Where they have their own namespace. Might be a good idea to update it? not blocker but seems odd that the other have it and the copy one doesn't

Reply:
I think it makes sense to remove their own namespace. There is no use of it. Command tasks have their own namespace, but belongs to different directory as well. 

wdyt @miguelsorianod ?

Reply:
Both approaches look good to me so up to you. In any case I think this would be a refactor that impacts all of them so we can do it in another PR/issue. To me this can be merged